Ontology/Space/Room/NurseryRoom.json

```diff
@@ -0,0 +1,9 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:building:NurseryRoom;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Nursery room"
+  },
+  "extends": "dtmi:digitaltwins:rec_3_3:core:Room;1",
```

Comment:
Should this extend from **HealthcareRoom**?

Reply:
My lack of English language is probably showing, but the translations of "Nursery" that I've found encompass both such rooms that would be found in hospitals, and rooms one might find in a private residence (i.e., roughly synonymous with "kid's room"). So I thought it was a more generic concept than only the hospital context. Am I mistaken?

Reply:
See: https://en.wikipedia.org/wiki/Nursery_(room)

Reply:
You are correct that it gets used in a residential context as well in the US. In a residential context, parents typically fitout a bedroom with furniture, etc. for the baby. I would still classify these as Bedrooms because this fitout is only temporary until the baby gets older. It would always be a Bedroom in terms of its classification at a residence.

I think this is a good link which defines the different types (levels) of a Neonatal Nursing Rooms:
https://en.wikipedia.org/wiki/Neonatal_nursing

Maybe we update to be NeonatalNursingRoom and have it extend from HealthcareRoom to remove any ambiguity?

Reply:
Sounds good, I'll fix this shortly!